vowpalwabbit/cb_explore_adf.cc

```diff
@@ -219,6 +219,8 @@ void get_cost_ranges(std::vector<float> &min_costs,
 
 size_t fill_tied(cb_explore_adf& data, v_array<action_score>& preds)
 {
+  if (preds.size() == 0 )
```

Comment:
It fixes the segfault issue, but it looks to me that the counting of examples would not be correct anymore?